### DISCUSSION TITLE
🤖 bots sending messages ?

### DISCUSSION THREAD

**Discussion by vicevolf** [misskey-dev/misskey#8553] [Q&A]
Is there any bot for this, or documentation for the api interface?

I want some automated service to link the bot to send notes.  For example using n8n to webhook to misskey?

**Comment by ThatOneCalculator** [comment]
You're going to have to code your own bot logic, but here's [the JS library](https://github.com/misskey-dev/misskey.js) and [the (unofficial) Python library](https://github.com/yupix/Mi.py)

**Comment by syuilo** [comment]
FYI: there is a bot implementation https://github.com/syuilo/ai
